deploy/Makefile

```
@@ -77,3 +92,100 @@ uninstall: ## Remove the release
+#   SMOKE_CLUSTER  cluster name                     (default omni-smoke)
+#   SMOKE_TIMEOUT  helm --wait timeout              (default 10m)
+#   SMOKE_PORT     host port for the /health check  (default 18882)
+#   SMOKE_SET      extra helm args, e.g. failure-injection overrides
+# -----------------------------------------------------------------------------
+SMOKE_CLUSTER ?= omni-smoke
+SMOKE_TIMEOUT ?= 10m
+SMOKE_PORT    ?= 18882
+SMOKE_SET     ?=
+
+.PHONY: deploy-smoke
+deploy-smoke: ## End-to-end self-host install on a disposable k3d/kind cluster
```

checkmake (0.3.2)

[warning] Target body for "deploy-smoke" exceeds allowed length of 5 lines (59).

(maxbodylength)